scripts/installs/install_devkit.bat

```diff
@@ -0,0 +1,8 @@
+mkdir "C:\Program Files\Rails_Server"
+mkdir "C:\Program Files\Rails_Server\devkit"
+powershell -Command "(New-Object System.Net.WebClient).DownloadFile('http://dl.bintray.com/oneclick/rubyinstaller/DevKit-mingw64-64-4.7.2-20130224-1432-sfx.exe', 'C:\Program Files\Rails_Server\devkit\devkit.exe')" <NUL
```

Comment:
If we're downloading devkit here, why do we have a file included in the commit for devkit? Are they two separate things?